frontend/app-development/layout/Layout.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Outlet, matchPath, useLocation } from 'react-router-dom';
+import { PageHeader } from './PageHeader';
+
+/**
+ * Displays the layout for the app development pages
+ */
+export const Layout = (): React.ReactNode => {
+  const { pathname } = useLocation();
+  const match = matchPath({ path: '/:org/:app', caseSensitive: true, end: false }, pathname);
+  const { org, app } = match.params;
+
```

Comment:
The Codecov comments in Layout will be tested in the #11444 issue as the logic currently being done in App.tsx will be handled in here.